wsl.go

```diff
@@ -257,9 +271,17 @@ func (p *Processor) parseBlockStatements(statements []ast.Stmt) {
 			leftHandSide                = p.findLHS(stmt)
 			rightHandSide               = p.findRHS(stmt)
 			rightAndLeftHandSide        = append(leftHandSide, rightHandSide...)
+			checkingNilErr              = p.isCheckingErrAgainstNil(stmt, leftHandSide, rightHandSide)
```

Comment:
There's a combined variable with both lhs and rhs you can use to minimize passed variables and to avoid the double call in `isCheckingErrAgainstNil`; `isCheckingErrAgainstNil`. It's assigned on the line above this. 😄

Reply:
I had used that originally but since I was enforcing that `nil` was also checked I wanted to explicitly check both lists to ensure that both were present in at least one. Now that we're removing the `nil` check I'll switch to using the var that includes both.